src/EnergyPlus/EarthTube.hh

```diff
@@ -85,7 +93,7 @@ namespace EarthTube {
         Real64 FanPressure = 0.0;
         Real64 FanEfficiency = 0.0;
         Real64 FanPower = 0.0;
-        Real64 GroundTempz1z2t = 0.0; // ground temp between z1 and z2 at time t
+        Real64 GroundTempt = 0.0; // ground temp at the depth of the earth tube midpoint at time t
```

Comment:
Lead us not to GroundTempt-ation.

Reply:
🤣🤣🤣...great comment! Yeah, I noticed the "tempt" part and chucked before just deciding that it was at least consistent with what the old name was.  Well, maybe it should have been GroundTempzt then...but since it was a single temperature and I wanted that to be very clear.